tests/testthat/test-helpers_design.R

```diff
@@ -99,6 +99,34 @@ test_that("h_add_dlts works as expected when first separate patient does not hav
   expect_true(data@nObs + 3 == result@nObs)
 })
 
+
+test_that("h_simulations_ouptput_format returns object as expected", {
```

Comment:
```suggestion
test_that("h_simulations_output_format returns object as expected", {
```
etc please check if typo occurs elsewhere and fix it